src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/HttpVariableDeclarationNode.cs

```diff
@@ -17,7 +17,7 @@ internal HttpVariableDeclarationNode(SourceText sourceText, HttpSyntaxTree? synt
     {
     }
 
-    public string VariableName => ChildTokens.Where(t => t.Kind == HttpTokenKind.Word).Select(t => t.Text).FirstOrDefault() ?? string.Empty;
+    public string VariableName => string.Join("", ChildTokens.Where(t => t.Kind == HttpTokenKind.Word || (t.Kind == HttpTokenKind.Punctuation && t.Text is "_")).Select(t => t.Text)) ?? string.Empty;
```

Comment:
Looks like this could return the wrong thing if there happened to be an `_` in the value portion of a variable declaration. Could you please add a test for that? (Again, I think having a dedicated sub node for the variable name may make things easier.)

Reply:
Ah never mind, I was confused by the naming. I assumed the `HttpVariableDeclarationNode` encompasses the entire declaration (inlcuding the `=` and the expression on the RHS). Looks like that is the `HttpVariableDeclarationAndAssignmentNode` :)

Since `HttpVariableDeclarationNode` only includes the name portion in addition to the `@` prefix, perhaps we could just take `this.Text` (which will strip away any leading and trailing trivia) and then strip out the first `@` character.

In other words, would something like below work?

```suggestion
    public string VariableName => this.Text.SubString(1)
```